docs/src/main/asciidoc/Installation_Guide.adoc

```diff
@@ -86,16 +86,41 @@ provision customized WildFly server using Galleon.
 Started Developing Applications Guide] shows you how to build Jakarta EE
 applications and deploy them to WildFly.
 
+[[WildFly_Maven_Plugin_Provisioning]]
+= Installing WildFly with the WildFly Maven Plugin
+
+The link:https://github.com/wildfly/wildfly-maven-plugin[WildFly Maven Plugin] exposes the `package` goal that allows you 
```

Comment:
Again we should like to the documentation here probably.

Reply:
Thanks, done.